lib/runners/htmlcs.js

```diff
@@ -48,13 +48,15 @@ runner.run = async options => {
 	 * @returns {void} Returns nothing.
 	 */
 	function configureHtmlCodeSniffer() {
-		if (options.rules.length && options.standard !== 'Section508') {
-			for (const rule of options.rules) {
-				if (window.HTMLCS_WCAG2AAA.sniffs.includes(rule)) {
-					window[`HTMLCS_${options.standard}`].sniffs[0].include.push(rule);
-				} else {
-					throw new Error(`${rule} is not a valid WCAG 2.0 rule`);
-				}
+		if (!options.rules.length || options.standard === 'Section508') {
```

Comment:
we removed Section 508 as a ruleset option in the 6.0 release. Though can the option can still be passed through to pa11y from Pa11y-CI?

Reply:
Hmmm I just did this to remove a lint warning. As we filter the rules below based on `HTMLCS_WCAG2AAA` anyway, I think we can drop this.